test/integration/conftest.py

```diff
@@ -116,3 +116,10 @@ def docker_image(docker_base_name, tag):
 def ecr_image(account_id, docker_base_name, tag, region):
     return '{}.dkr.ecr.{}.amazonaws.com/{}:{}'.format(
         account_id, region, docker_base_name, tag)
+
+
+@pytest.fixture(autouse=True)
+def skip_py2_containers(request, tag):
```

Comment:
there is a fixture for the Python version: https://github.com/aws/sagemaker-tensorflow-container/blob/master/test/integration/utils.py#L30-L32

if you want to use it, you could define this `skip_py2_containers` fixture to the `test_experiments` file

Reply:
removed the fixture